src/index.html

```diff
@@ -1,13 +1,409 @@
+            >
+              <img
+                src="images/photos/picture2.png"
+                alt="picture 2"
+                class="pictures__photo"
+              >
+            </div>
+          </div>
+          <div>
+            <h4 class="page__text-red">A New Look at Old Masters</h4>
+            <p>December 12, 2020 - Through Spring 2023</p>
+          </div>
+        </div>
+
+        <div class="grid__bottom pictures">
+          <div class="pictures__bottom grid__squares">
+            <img
+              src="images/photos/picture3.png"
+              alt="picture 3"
+              class="pictures__photo"
+            >
+            <img
+              src="images/photos/picture4.png"
+              alt="picture 4"
+              class="pictures__photo"
+            >
+            <img
+              src="images/photos/picture5.png"
+              alt="picture 5"
+              class="pictures__photo pictures__photo--hide"
+            >
+          </div>
+          <div>
+            <h4 class="page__text-red">Bodhisattvas of Wisdom, Compassion, and Power</h4>
+            <p>March 27, 2021 - October 30, 2022</p>
+          </div>
+        </div>
+
+        <div class="grid__bottom-last pictures">
+          <div class="pictures__bottom grid">
+            <div
+              class="grid__item grid__item--1-1 grid__item--tablet-1-2 grid__item--desktop-1-4"
+            >
+              <img
+                src="images/photos/picture6.png"
+                alt="picture 6"
+                class="pictures__photo"
+              >
+            </div>
+            <div
+              class="grid__item grid__item--2-2 grid__item--tablet-3-6 grid__item--desktop-5-12"
+            >
+              <img
+                src="images/photos/picture7.png"
+                alt="picture 7"
+                class="pictures__photo pictures__photo--right"
+              >
+            </div>
+          </div>
+          <div>
+            <h4 class="page__text-red">Inspiring Walt Disney: The Animation of French Decorative Arts</h4>
+            <p>December 10, 2021 - March 6, 2022</p>
+          </div>
+        </div>
+
+        <div class="grid grid--desktop">
+          <div
+            class="grid__item grid__item--1-2 grid__item--tablet-2-5 grid__item--desktop-5-8"
+          >
+            <a href="#explore" class="button button__link">View all exhibitions</a>
+          </div>
+        </div>
+      </section>
+      <section class="page__section" id="explore">
+        <div class="grid grid--explore">
+          <div
+            class="grid__item grid__item--1-2 grid__item--tablet-2-5 grid__item--desktop-4-9"
+          >
+            <h2 class="page__section-title">More to Explore Online</h2>
+          </div>
+
+          <div
+            class="grid__item grid__item--1-2 grid__item--tablet-2-5 grid__item--desktop-1-4"
+          >
```

Comment:
only in one column on 1200px

Reply:
I've just got the mockup breakpoints wrong)